config/module/tree_test.go

```diff
@@ -200,18 +216,6 @@ func TestTreeName(t *testing.T) {
 	}
 }
 
-func TestTreeValidate_badChild(t *testing.T) {
-	tree := NewTree("", testConfig(t, "validate-child-bad"))
-
-	if err := tree.Load(testStorage(t), GetModeGet); err != nil {
-		t.Fatalf("err: %s", err)
-	}
-
-	if err := tree.Validate(); err == nil {
-		t.Fatal("should error")
-	}
-}
-
```

Comment:
Removed this test and the test below since the validation catches these errors earlier now.